superset-frontend/src/views/CRUD/data/query/QueryList.tsx

```diff
@@ -325,7 +318,13 @@ function QueryList({ addDangerToast, addSuccessToast }: QueryListProps) {
         }: any) => (
           <Tooltip title={t('Open query in SQL Lab')} placement="bottom">
             <a href={`/superset/sqllab?queryId=${id}`}>
-              <Icon name="full" />
+              <Icons.Full
```

Comment:
`iconColor`?

Reply:
changing. I did this becasue I thought adding useTheme would break the tests for the querylist but it doesn't look like it.

Reply:
changing. I did this becasue I thought adding useTheme would break the tests for the querylist but it doesn't look like it.